packages/truffle-debugger/test/data/calldata.js

```diff
@@ -180,6 +188,40 @@ describe("Calldata Decoding", function() {
     assert.include(variables, expectedResult);
   });
 
+  it("Decodes dynamic structs correctly", async function() {
+    this.timeout(6000);
+    let instance = await abstractions.CalldataTest.deployed();
+    let receipt = await instance.stringBoxTest({ it: "hello world" });
+    let txHash = receipt.tx;
+
+    let bugger = await Debugger.forTx(txHash, {
+      provider,
+      files,
+      contracts: artifacts
+    });
+
+    let session = bugger.connect();
+
+    let sourceId = session.view(solidity.current.source).id;
+    let source = session.view(solidity.current.source).source;
```

Comment:
Can skip the doubles here and do:
```javascript
let { source, id: sourceId } = session.view(solidity.current.source)
```

Reply:
(also it looks like all the `let`s can just be `const`s)

Reply:
Yeah, that's a bunch of copypaste that I think I'm going to skip changing for now.  I can redo these more generally later.  But thanks for pointing it out.